src/main/scala/afenton/bazel/bsp/BazelBspApp.scala

```diff
@@ -72,7 +75,8 @@ object BazelBspApp
 
   def main: Opts[IO[ExitCode]] =
     (verboseOpt, verifySetupOpt, setupOpt).mapN { (verbose, verify, setup) =>
-      if setup then writeBspConfig(BuildMetaData.Version).as(ExitCode.Success)
+      if setup then
+        writeBspConfig(Paths.get("").toAbsolutePath).as(ExitCode.Success)
```

Comment:
nit:

```scala
val currentDir: IO[Path] = IO.blocking(Paths.get("").toAbsolutePath)
```
that is a side effect.

or:
```scala
for {
  cwd <- IO.blocking(Paths.get("").toAbsolutePath)
  _ <- writeBspConfig(cwd)
} yield ExitCode.Success
```
which is maybe a bit more verbose, but I think for-comprehension is very readable for IO heavy code.